packages/agents/watcher/src/config.ts

```diff
@@ -52,15 +52,27 @@ export const getEnvConfig = (): WatcherConfig => {
     process.exit(1);
   }
   // return configFile;
+  const parsedChains: object = process.env.WATCHER_CHAIN_CONFIG
+    ? JSON.parse(process.env.WATCHER_CHAIN_CONFIG)
+    : configJson.chains
+    ? configJson.chains
+    : configFile.chains;
+
+  const chains: any = {};
+  Object.entries(parsedChains).map(([key, values]) => {
```

Comment:
You made a sep. sanitized `chains` object but didn't end up using it? It's using `parsedChains` below

Reply:
i fix